common/tools/dev-tool/package.json

```diff
@@ -12,14 +12,12 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build": "tsc",
     "build:test": "echo Skipped.",
-    "clean": "rimraf dist dist-* *.tgz *.log",
```

Comment:
why deleting the `clean` script here?

Reply:
since that clears out the *.tgz and the log files that are useful for CI

Reply:
It can still be useful for devs locally.

Reply:
and removing it means `rush clean` will error because there is no `clean` script for this package.

Reply:
@deyaaeldeen added it back. Let me know when it's safe to merge